content/learn/getting-started/npm.en.md

```diff
@@ -55,7 +55,9 @@ Shorthands of the flags can also be used:
 - \-D: `--save-dev`
 - \-O: `--save-optional`
 
-The difference between _devDependencies_ and _dependencies_ is that the former contains development tools, like a testing library, while the latter is bundled with the app in production.
+The difference between _dependencies_ and _devDependencies_ is that _dependencies_ are required to run for the production, _devDependencies_ only during development.
+_dependencies_ are installed transitively: if A requires B, and B requires C, then C gets installed, otherwise, B could not work, and neither would A.
+_devDependencies_ is not installed transitively. E.g. we don't need to test B to test A, so B's testing dependencies can be left out.
```

Comment:
Could you quote from where this technical detail comes from?

I also would disagree regarding the explanation you gave of dependencies vs devDependencies. As "development" and "production" are verbatim.

You should probably emphasize that "devDependencies" are not required for a working-version of the App and should only be used for local-development, tooling, testing and code coverage. Whereas, anything imported in any line of actual code that runs in a built/bundled/packaged version of the code should be on "dependencies".